Automattic/jetpack
# Update/videopress convert old embed blocks

Fixes Automattic/greenhouse#1058

#### Changes proposed in this Pull Request:

This PR adds a new Gutenberg transform matching conversion from classic embed for v.wordpress.com to a core/embed block using videopress.com/v/ URL.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* On a Jetpack site with VideoPress enabled
* Add a classic block and edit it as HTML
* Paste the following embed code
`<embed src="http://v.wordpress.com/KllQxFVq"  type="application/x-shockwave-flash" width="600" height="338"  allowscriptaccess="always" allowfullscreen="true">`
* Click on "Edit visually"
* Click on "Convert to block"
* ✅ A core/embed block should be created, using the VideoPress player, and the video should work.
![Capture d’écran 2021-12-29 à 16 00 01](https://user-images.githubusercontent.com/1420594/147675255-0daefc20-f540-4ddf-93a7-e6e14021c2d1.png)

#### Notes

Maybe we could also propely handle v.videopress.com link embed, that seems to require some other stuff to be done, so I'll create a new issue.
